core/bin/external_node/src/config/mod.rs

```diff
@@ -66,7 +71,12 @@ impl RemoteENConfig {
                 .context("Failed to fetch L1 chain ID")?
                 .as_u64(),
         );
-
+        let base_token_addr = match client.get_base_token_l1_address().await {
+            Err(ClientError::Call(err)) if ErrorCode::MethodNotFound.code() == err.code() => {
+                Address::from_str("0x0000000000000000000000000000000000000001")?
```

Comment:
Nit: You can use `"...".parse()` instead of importing `FromStr`; ditto in other places. Also, I'd suggest to create a constant for the default (fallback?) base token L1 address (probably somewhere in the `zksync_types` crate) so that there's no risk that it's mismatched in various places it's used.

Reply:
Done, this is now a constant over on  lib/constants/src/contracts.rs file